packages/imageviewer-extension/src/index.ts

```diff
@@ -215,15 +215,15 @@ function addCommands(app: JupyterLab, tracker: IImageTracker) {
     const widget = tracker.currentWidget;
 
     if (widget) {
-      widget.rotation += 90;
+      widget.rotation -= 90;
```

Comment:
Don't we want "clockwise" to still mean clockwise, and instead switch which command is tied to the `[` key? I think this change makes the clockwise and counter-clockwise commands do the opposite of what they advertise.